app/routes/about-us/route.tsx

```diff
@@ -1,3 +1,22 @@
+import styles from './about-us.module.scss';
+
 export default function AboutUsPage() {
-    return <h1>About us</h1>;
+    return (
+        <div className={styles.root}>
+            <div className={styles.aboutSection}>
+                <h1 className={styles.header1}>We are ReClaim</h1>
+                <p className={styles.header2}>A women-owned local business</p>
```

Comment:
Call these `title` and `subtitle`. Because we have common styles called `heading1` and `heading2`, and it's too confusing when the styles don't match those.